packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -247,11 +247,12 @@ namespace Ifpack2 {
       template<typename T, int l> using Vector = KokkosBatched::Experimental::Vector<T,l>;
       template<typename T> using SIMD = KokkosBatched::Experimental::SIMD<T>;
       template<typename T, typename M> using DefaultVectorLength = KokkosBatched::Experimental::DefaultVectorLength<T,M>;
+      template<typename T, typename M> using DefaultInternalVectorLength = KokkosBatched::Experimental::DefaultInternalVectorLength<T,M>;
```

Comment:
If the alias has the same number of template parameters as the original, you could just say `using KokkosBatched::Experimental::DefaultInternalVectorLength;`.  This is not incorrect, though.

Reply:
I use this aliasing syntax after I experience a problem from a certain compiler a long time ago. Probably the problem is gone now. It is just my habit. I am working on improvement and preparing for the second PR and I will test the simple aliasing syntax and include the modification.